homeassistant/components/ecobee/const.py

```diff
@@ -9,4 +9,18 @@
 CONF_INDEX = "index"
 CONF_REFRESH_TOKEN = "refresh_token"
 
+ECOBEE_MODEL_TO_NAME = {
+    "idtSmart": "ecobee Smart",
+    "idtEms": "ecobee Smart EMS",
+    "siSmart": "ecobee Si Smart",
+    "siEms": "ecobee Si EMS",
+    "athenaSmart": "ecobee3 Smart",
+    "athenaEms": "ecobee3 EMS",
+    "corSmart": "Carrier/Bryant Cor",
+    "nikeSmart": "ecobee3 lite Smart",
+    "nikeEms": "ecobee3 lite EMS",
+}
+
 ECOBEE_PLATFORMS = ["binary_sensor", "climate", "sensor", "weather"]
+
+MANUFACTURER = "ecobee"
```

Comment:
Capitalize ?

Reply:
It's stylized as "ecobee" (no capital) everywhere, on the website, packaging, app, etc.